engine/samples/input/main.cpp

```diff
@@ -138,6 +138,35 @@ static void showcaseUnbound(const Window& window, bool& explained)
 }
 /// [Showcase Unbound]
 
+/// [Showcase Mouse Buttons]
```

Comment:
This is a reference to be used in the sample docs page. Don't forget to add this section to the page too! Check out `engine/samples/input/page.md`